services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy

```diff
@@ -1105,6 +1105,60 @@ class ProtectedPathSpec extends HealthCheckSpecification {
         flowHelper.deleteFlow(flow.id)
     }
 
+    @Ignore("https://github.com/telstra/open-kilda/issues/2762")
+    def "System doesn't mark protected path as DOWN when can't find non overlapping protected path during intentional\
```

Comment:
Sorry? The system _SHOULD_ mark protected path as DOWN when it can't find non overlapping protected path